lang/cs/Org.Apache.REEF.Bridge.Core.Common/Org.Apache.REEF.Bridge.Core.nuspec

```diff
@@ -0,0 +1,31 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is this file needed? You should be able to just use the csproj unless there are specific custom steps that you want to use for producing the nuget.